blackbox/docs/admin/system-information.rst

```diff
@@ -1826,6 +1826,71 @@ node.
+Lucene file structures for troubleshooting and performance optimization
+of shards.
+
++-------------------+-------------------------------------------+-------------+
+| Column Name       | Description                               | Return Type |
++===================+===========================================+=============+
+| ``table_schema``  | Schema name of the table of the shard.    | ``TEXT``    |
++-------------------+-----------+-------------------------------+-------------+
+| ``table_name``    | Table name of the shard.                  | ``TEXT``    |
++-------------------+-----------+-------------------------------+-------------+
+| ``node_id``       | ID of the node on which the shard resides.| ``TEXT``    |
++-------------------+-----------+-------------------------------+-------------+
+| ``shard_id``      | ID of the effected shard.                 | ``INTEGER`` |
++-------------------+-----------+-------------------------------+-------------+
+|``segment_name``   | Name of the segment, derived from the     | ``TEXT``    |
+|                   | segment generation and used internally    |             |
+|                   | to create file names in the directory of  |             |
+|                   | the shard.                                |             |
++-------------------+-----------+-------------------------------+-------------+
+| ``generation``    | Generation number of the segment,         |``INTEGER``  |
+|                   | increments for each segment written.      |             |
++-------------------+-----------+-------------------------------+-------------+
+| ``num_docs``      | Number of non-deleted lucene documents    | ``INTEGER`` |
+|                   | in the segment.                           |             |
++-------------------+-----------+-------------------------------+-------------+
+| ``deleted_docs``  | Number of deleted lucene documents in the | ``INTEGER`` |
+|                   | segment.                                  |             |
++-------------------+-----------+-------------------------------+-------------+
+| ``size``          | Disk space used by the segment in bytes.  | ``INTEGER`` |
++-------------------+-----------+-------------------------------+-------------+
+| ``memory``        | Segment data stored in memory for         | ``INTEGER`` |
+|                   | efficient search, -1 if it is unavailable.|             |
++-------------------+-----------+-------------------------------+-------------+
+| ``committed``     | Indicates if the segments are synced to   | ``BOOLEAN`` |
+|                   | disk. Segments that are synced can survive|             |
+|                   | a hard reboot.                            |             |
++-------------------+-----------+-------------------------------+-------------+
+| ``search``        | Indicates if the segment is searchable.   | ``BOOLEAN`` |
```

Comment:
I think this is useful information: `If false, the segment has most likely been written to disk but needs a refresh to be searchable.`